app/src/components/layout/errorBox/ErrorBox.tsx

```diff
@@ -44,6 +47,17 @@ const ErrorBox = ({
                             </ErrorSummary.Item>
                         )}
                         {messageBody && <p>{messageBody}</p>}
+                        {errorMessageList &&
+                            errorMessageList.map((errorItem) => {
```

Comment:
sonar cloud says it would prefer just errorMessageList?.map instead of errorMessageList && errorMessageList.map